tests/unit/config/test_review_config.py

```
@@ -63,6 +64,58 @@ def test_load_config_defaults_review_section_when_missing(
+
+    config = load_config(config_path=config_file)
+
+    assert_that(config.review.depth).is_equal_to(2)
+    assert_that(config.review.force_semantic_chunking).is_true()
+
+
+def test_review_config_rejects_invalid_depth() -> None:
+    """Review depth must be between 1 and 3."""
+    from lintro.config.review_config import ReviewConfig
+
+    with pytest.raises(ValueError):
```

Ruff (0.15.20)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)